src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -1278,6 +1265,26 @@ export class BrightScriptDebugSession extends BaseDebugSession {
         deferred.resolve();
     }
 
+    private async evaluateTemporaryVariables(args: DebugProtocol.EvaluateArguments, variablePath: string[]): Promise< { evalArgs: DebugProtocol.EvaluateArguments; variablePath: string[] } > {
```

Comment:
I'm wondering if a better name for this would be something like `evaluateExpressionToTempVar`? Thoughts?